addons/api/addon/serializers/credential.js

```diff
@@ -28,13 +28,17 @@ export default class CredentialSerializer extends ApplicationSerializer {
     // Remove non-username_password type attributes
     delete serialized.attributes.private_key;
     delete serialized.attributes.passphrase;
+    // Remove password from the payload if null
+    if (!serialized?.attributes?.password)
```

Comment:
Is this safe? Can someone be allowed to set an empty string as their password? Can the field both be null or undefined or always just one? Curious if we can make these check be a little more specific.

Reply:
Users cannot unset password.  Probably could add a comment here to that effect.

Reply:
Correct. I can add a comment to explain that here. If the user wants to remove that credential password, they have to delete the whole thing.